lib/postcode_validation.rb

```diff
@@ -3,7 +3,10 @@
 module PostcodeValidation
   require 'httparty'
   require 'postcode_validation/domain/potential_address_match'
+  require 'postcode_validation/domain/address_list'
```

Comment:
Sadly we couldn't use just the gateway in our postcode service since it hits the wrong endpoint in PCA.  This evolved into all the components you see here.